src/mapdraw.c

```diff
@@ -1106,7 +1106,8 @@ int msDrawVectorLayer(mapObj *map, layerObj *layer, imageObj *image) {
 
   /* build item list. STYLEITEM javascript needs the shape attributes */
   if (layer->styleitem &&
-      (strncasecmp(layer->styleitem, "javascript://", 13) == 0)) {
+      ((strncasecmp(layer->styleitem, "javascript://", 13) == 0) ||
```

Comment:
for your information, if you include "cpl_port.h" from GDAL, you can juste use ``STARTS_WITH_CI(layer->styleitem, "javascript://") || lSTARTS_WITH_CI(layer->styleitem, "sld://")`` which saves you from a potential wrong string length

Reply:
I've updated the changes in this pull request to use `STARTS_WITH_CI` (note to self - CI is for Case Insensitive). There are plenty of places in the MapServer codebase that could also switch to this approach.